src/sections/Blog-single/index.js

```diff
@@ -6,7 +6,7 @@ import slugify from "../../utils/slugify";
 
 import { Container, Row, Col } from "../../reusecore/Layout";
 import PageHeader from "../../reusecore/PageHeader";
-import Sidebar from "../Blog-sidebar";
+import RelatedPosts from "../Related-Posts";
 
 import BlogPageWrapper from "./blogSingle.style";
 
```

Comment:
It seems like this page doesn't require a graphql query anymore, right?
@OmkarPh , @Tanuj22

Reply:
Yes it can be removed

Reply:
Yes it can be removed

Reply:
I don't know what that query in Blog-single does, so I didn't touch it :)

Reply:
Is there anything else to change?

Reply:
Remove this query